packages/scripts/scripts/test-script.js

```diff
@@ -1,7 +1,6 @@
 // Do this as the first thing so that any code reading it knows the right env.
 process.env.BABEL_ENV = 'test';
 process.env.NODE_ENV = 'test';
-process.env.PUBLIC_URL = '';
```

Comment:
Unrelated change, this shouldn't be in this PR, it should be updated by removing it in #62